waiter/integration/waiter/health_check_test.clj

```diff
@@ -85,6 +86,21 @@
       (run-ping-service-test waiter-url idle-timeout command backend-proto health-check-proto num-ports health-check-port-index
                              :query-params {"exclude" "service-state"}))))
 
+(deftest ^:parallel ^:integration-fast test-invalid-backend-proto-health-check-proto-combo
+  (testing-using-waiter-url
+    (let [supported-protocols #{"http" "https" "h2c" "h2"}]
+      (for [backend-proto supported-protocols
```

Comment:
```suggestion
      (doseq [backend-proto supported-protocols
```

Reply:
Sorry, I should have been more clear: the 3 `doseq`s can be combined onto one `doseq` (not changed to a lazy `for`).